src/PostmarkTransport.php

```diff
@@ -29,11 +34,11 @@ class PostmarkTransport extends Transport
     protected $key;
 
     /**
-     * The Postmark API end-point.
+     * The Postmark API endpoint.
      *
      * @var string
      */
-    protected $url = 'https://api.postmarkapp.com/email';
+    protected $apiEndpoint = self::API_ENDPOINT;
```

Comment:
I don't think we need to have both a constant and an instance variable do we? Can we just leave the string here or use the constant instead?

Reply:
Sure, but we are altering the endpoint url based on a condition. In this way we will always have a reference to the base endpoint, which will never change, so I declared this as a constant.

Reply:
Oh I just noticed you aren't returning inside the if statement. I'm ok with this then 👍